src/providers/index.ts

```diff
@@ -49,10 +51,14 @@ const Providers: { [key: string]: ProviderConfigs } = {
   jina: JinaConfig,
   'fireworks-ai': FireworksAIConfig,
   'workers-ai': WorkersAiConfig,
+<<<<<<< HEAD
```

Comment:
Seems like the merge conflicts were not resolved properly. Can you please check this?